x-pack/plugins/triggers_actions_ui/public/application/components/button_group_field.tsx

```diff
@@ -60,16 +62,25 @@ const ButtonGroupFieldComponent: React.FC<Props> = ({
             isInvalid={isInvalid}
             fullWidth
           >
-            <EuiButtonGroup
-              isFullWidth
-              buttonSize="m"
-              legend="Select"
-              color="primary"
-              options={[]}
-              idSelected={field.value}
-              onChange={field.setValue}
-              {...rest}
-            />
+            {isButton ? (
```

Comment:
The form lib library supports the `EuiSelect` field. You can use it as `<UseField component={SelectField} ...>` in your component. If you want to create a `EuiSelect` field specialized to your needs (based on your code it does not seem so) then it is better if you create another component and not modify this one as it is dedicated for `EuiButtonGroup`.